ext/standard/var.c

```diff
@@ -36,13 +36,28 @@ struct php_serialize_data {
 
 #define COMMON (is_ref ? "&" : "")
 
+static void php_print_escaped(zend_string *str) {
```

Comment:
Naming nit; "escaped" is imprecise and implies much more escaping than what this does. I'd prefer one of:

- `php_var_dump_escaped` - This says it's specifically var_dump escaping, which can be understood as only effecting null bytes.
- `php_print_escape_null` - Self explanatory, really

I know this is static linkage and won't leak, but var.c is a busy place, and the potential for another place to call a function that and be human-ambiguous (even if not compiler-ambiguous) is real.